### PR TITLE
Abhijit_Assignment_9_Done

### PR DESCRIPTION
**Pull request by AbhijitMJun20**
Abhijit_Assignment_9_Done

### CODE REVIEW

File: src/abhijitMJun20/Staticnonstatic.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package abhijitMJun20;
+
+public class Staticnonstatic { 
+	static double sumofvalues;
```
Comment (KrishnaKTechnocredits) [inline src/abhijitMJun20/Staticnonstatic.java]: Abhijit, have you read assignment properly.
Rule D) Class should not have any instance variable. Remove instance variables please.

File: src/abhijitMJun20/Staticnonstatic.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	static double average;
+	double sum( double parameter1, double parameter2,double parameter3){
+		 sumofvalues=parameter1+parameter2+parameter3;
+		return sumofvalues;
+	}
+	double average(double parameter4, double parameter5,double parameter6){  
+	       sum(parameter4,parameter5,parameter6);
```
Comment (KrishnaKTechnocredits) [inline src/abhijitMJun20/Staticnonstatic.java]: int sumOfValues =  sum(parameter4,parameter5,parameter6);

File: src/abhijitMJun20/Staticnonstatic.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+		 return "A Grade";
+		 }
+	 else {
+		 return "B Grade";
+		 }
+	}
+	public static void main(String[] args)
+	{
+		Staticnonstatic staticnonstatic= new Staticnonstatic();
+		staticnonstatic.average(10,20,50000);
```
Comment (KrishnaKTechnocredits) [inline src/abhijitMJun20/Staticnonstatic.java]: average = staticnonstatic.average(10,20,50000);

File: src/abhijitMJun20/Staticnonstatic.java (hunk 1 above)
Comment (AbhijitMJun20) [reply src/abhijitMJun20/Staticnonstatic.java]: Done

File: src/abhijitMJun20/Staticnonstatic.java (hunk 2 above)
Comment (AbhijitMJun20) [reply src/abhijitMJun20/Staticnonstatic.java]: Done

File: src/abhijitMJun20/Staticnonstatic.java (hunk 3 above)
Comment (AbhijitMJun20) [reply src/abhijitMJun20/Staticnonstatic.java]: Done

**Review by KrishnaKTechnocredits — COMMENTED** [review]
Code looks good to me.

**Pull request merged**